ichnaea/taskapp/app.py

```diff
@@ -13,6 +19,15 @@
 )
 
 
+@setup_logging.connect
+def setup_logging_process(loglevel, logfile, format, colorize, **kwargs):
```

Comment:
I would guess that ``loglevel`` is the ``--loglevel`` passed from the command line. If that's the case, we could configure the log level from ``scripts/run_celery.sh`` and ``scripts/run_scheduler.sh``. I'd support waiting until will configure from an environment variable to take that step.

Reply:
I forgot we set it with a flag there. I'll nix the `--loglevel` argument in the `run_*` scripts.